exercises/practice/pangram/pangram_test.py

```diff
@@ -40,8 +40,8 @@ def test_missing_letters_replaced_by_numbers(self):
     def test_mixed_case_and_punctuation(self):
         self.assertIs(is_pangram('"Five quacking Zephyrs jolt my wax bed."'), True)
 
-    def test_case_insensitive(self):
-        self.assertIs(is_pangram("the quick brown fox jumps over with lazy FX"), False)
+    def test_a_m_and_a_m_are_26_different_characters_but_not_a_pangram(self):
```

Comment:
👀 while maybe not true snake case, using `A_M` for the second range may may more sense in context?

Reply:
🤔   Yeah.  Problem is that this is auto-generated, and I don't know if I can get the generator to fix it.  Let me try under a separate PR, if that's OK?